test/annotation_utils.t

```diff
@@ -117,23 +117,32 @@ my $test_ftrs = [{
 
 ## Re-mapping the contigIDs back to their original (long) names
 subtest '_remap_contigIDs' => sub {
-    my $contigID_hash = {
+    my $contigID_hash = {};
+    my $gn0 = {
+            contig_ids => ['fake_contig_id_1', 'fake_contig_id_2'],
+    };
+    my $gn1 = $annoutil->_remap_contigIDs($contigID_hash, $gn0);
+    cmp_deeply($gn1, $gn0, "No contigID mapping happened--contigID_hash was empty");
+
+    $contigID_hash = {
           'contigID_1' => 'NZ_CP028859.1',
-          'contigID_2' => 'NZ_CP028860.1'
+          'contigID_2' => 'NZ_CP028860.1',
+          'contigID_3' => 'NZ_CP028859.1',
+          'contigID_4' => 'CP0035411Candidatus_Carsonella_ruddii_CE_isolate_Thao2000_complete_genome'
     };
+    $gn0 = { contig_ids => [] };
+
+    $gn1 = $annoutil->_remap_contigIDs($contigID_hash, $gn0);
+    ok ( @{$gn1->{contig_ids}} eq 0, "No contigID remapping because of empty contig id array.");
+
     my $gn = {
-        contigs => [
-            {id => 'contigID_1',
-             name => 'contigID_1'},
-            {id => 'contigID_2',
-             name => 'contigID_2'}
-        ]
+           contig_ids => ['contigID_1', 'contigID_2', 'contigID_3', 'contigID_4']
     };
     $gn = $annoutil->_remap_contigIDs($contigID_hash, $gn);
-    my $contig1 = $gn->{contigs}[0];
-    my $contig2 = $gn->{contigs}[1];
-    ok ($contig1->{id} eq $contigID_hash->{contigID_1}, "mapped contigID correctly");
-    ok ($contig2->{id} eq $contigID_hash->{contigID_2}, "mapped contigID correctly");
+    ok ($gn->{contig_ids}->[0] eq $contigID_hash->{contigID_1}, "mapped contigID correctly");
+    ok ($gn->{contig_ids}->[1] eq $contigID_hash->{contigID_2}, "mapped contigID correctly");
+    ok ($gn->{contig_ids}->[2] eq $contigID_hash->{contigID_3}, "mapped contigID correctly");
+    ok ($gn->{contig_ids}->[3] eq $contigID_hash->{contigID_4}, "mapped contigID correctly");
```

Comment:
I'd write this as
```
cmp_deeply
    $gn->{ contig_ids },
    [  $contigID_hash->{ contigID_1 }, $contigID_hash->{ contigID_2 }, $contigID_hash->{ contigID_3 }, $contigID_hash->{ contigID_4 } ],
    'contig IDs remapped correctly';
```
so that all your tests in this area have the same format.

Reply:
Nice. It's neater.